README.md

```diff
@@ -1,16 +1,106 @@
-# test #
+# skeleton-golang-package #
 
-[![GitHub Build Status](https://github.com/cisagov/test/workflows/build/badge.svg)](https://github.com/cisagov/test/actions)
+[![GitHub Build Status](https://github.com/cisagov/skeleton-golang-package/workflows/build/badge.svg)](https://github.com/cisagov/skeleton-golang-package/actions)
 
 This is a generic skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) GitHub project started.
+new [cisagov](https://github.com/cisagov) Go project started.
 This skeleton project contains [licensing information](LICENSE), as
 well as [pre-commit hooks](https://pre-commit.com) and
 [GitHub Actions](https://github.com/features/actions) configurations
 appropriate for the major languages that we use.
 
-In many cases you will instead want to use one of the more specific
-skeleton projects derived from this one.
+## Installation ##
+
+To install the example package, use `go build`:
+
+```console
+cd ./skeleton-golang-package/src/example
```

Comment:
It is a fairly standard practice to make all references relative to the project's root. It's also one less place where a downstream repo has to worry about changing references from skeleton-golang-package to their repo name.

```suggestion
cd src/example
```